cmd/create/oidcconfig/cmd.go

```diff
@@ -195,18 +249,6 @@ func (s *CreateOidcConfigManualStrategy) execute(r *rosa.Runtime) (string, strin
 		AddParam(awscb.CreateBucketConfiguration, createBucketConfig).
```

Comment:
I think we should remove the `createS3BucketCommand` from the manual mode. Just output the discovery document and the private key, and let the customer host it wherever they want.

Reply:
I see your point, but I'd say to keep consistency we should keep it. And maybe add a --raw or something like that as a facilitator for people who don't want to deal with setting up the documents, but still want to host elsewhere

Reply:
Decided to set it up